.github/workflows/renovate.yml

```diff
@@ -14,17 +14,17 @@ jobs:
   setup:
     uses: ./.github/workflows/log_run_details.yml
   renovate:
-    if: github.repository == 'kubearchive/kubearchive'  # Don't run renovate on forks
+    if: github.repository == 'kronicler/kronicler'  # Don't run renovate on forks
     runs-on: ubuntu-latest
     steps:
-      - name: Get kubearchive-renovate Github App Api Token
+      - name: Get kronicler-renovate Github App Api Token
         id: get_token
         uses: actions/create-github-app-token@v1
         with:
-          private-key: ${{ secrets.KUBEARCHIVE_RENOVATE_PRIVATE_KEY }}
-          app-id: ${{ secrets.KUBEARCHIVE_RENOVATE_APP_ID }}
+          private-key: ${{ secrets.KRONICLER_RENOVATE_PRIVATE_KEY }}
+          app-id: ${{ secrets.KRONICLER_RENOVATE_APP_ID }}
```

Comment:
This secret will need to be to be renamed. I added a comment on the issue for this PR